packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -139,14 +135,12 @@ export function RepositoryMixin<T extends MixinTarget<Application>>(
       const options = toOptions(nameOrOptions);
       // We have an instance of
       if (dataSource instanceof juggler.DataSource) {
-        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-        const name = options.name || dataSource.name;
+        const name = options.name ?? dataSource.name;
```

Comment:
I believe we are intentionally using the operator `||` here, please revert this change.